app/assets/javascripts/discourse/app/lib/notification-items/base.js

```diff
@@ -84,14 +84,14 @@ export default class NotificationItemBase {
   /**
    * @returns {string[]} Include additional classes to the label's wrapper <span>.
    */
-  get labelWrapperClasses() {
+  get labelClasses() {
     return [];
   }
 
   /**
    * @returns {string[]} Include additional classes to the description's wrapper <span>.
```

Comment:
Does the description here need to be updated since it is no longer related to the wrapper?

Reply:
yes, done!